packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -59,7 +64,25 @@ const Radio = ({
   value,
   onChange: onChangeProp,
   onBlur: onBlurProp,
+  inputRef,
 }: RadioProps) => {
+  const ref = useRef(null);
+
+  useImperativeHandle(
+    inputRef,
+    () => {
+      const element = ref.current as unknown as HTMLElement;
+      const inputElement =
+        element.tagName === "INPUT" ? element : element.querySelector("input");
+      return {
+        focus: () => {
+          inputElement?.focus();
```

Comment:
Shouldn't it throw or stop earlier? The optional chaining postpones it until `.focus()` is called

Reply:
good catch! updated